proto/sedachain/pubkey/v1/query.proto

```diff
@@ -11,6 +11,11 @@ option go_package = "github.com/sedaprotocol/seda-chain/x/pubkey/types";
 
 // Query defines the gRPC querier service.
 service Query {
+  // Params returns the total set of pubkey parameters.
+  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+    option (google.api.http).get = "/cosmos/mint/params";
```

Comment:
This should probably be `"/sedachain/pubkey/params";`

Reply:
Whoops forgot to change that when looking at how Cosmos did this my bad 😅